### PR TITLE
Hide Python packages in show by default

### PR DESCRIPTION
**Pull request by bfirsh**
Lots of Python packages make `replicate show` hard to use.

If there are interesting packages (tensorflow, torch, etc), show them. Otherwise, show at most 5. If `--all` is passed, show them all.

Looks a bit like this:

Screen Shot 2021-01-13 at 19 27 34

Any other ideas for "interesting" Python packages?

### CODE REVIEW

File: go/pkg/cli/show.go [hunk 1]
```diff
@@ -161,7 +163,32 @@ func showExperiment(au aurora.Aurora, out io.Writer, proj *project.Project, exp
...
+		return true
+	case "tensorflow":
+		return true
+	case "torch":
+		return true
+	}
+	return false
+}
+
+func filter(ss []string, test func(string) bool) (ret []string) {
```
Comment (andreasjansson) [inline go/pkg/cli/show.go:182]: maybe move this to `slices.FilterStrings`?

File: go/pkg/cli/show.go [hunk 2]
```diff
@@ -161,7 +163,32 @@ func showExperiment(au aurora.Aurora, out io.Writer, proj *project.Project, exp
 	return nil
 }
 
-func writeExperimentCommon(au aurora.Aurora, w *tabwriter.Writer, exp *project.Experiment, experimentRunning bool) {
+func isInterestingPythonPackage(pkg string) bool {
```
Comment (andreasjansson) [inline go/pkg/cli/show.go:166]: add cntk, mxnet, keras and jax?

**Review by andreasjansson — APPROVED** [review]
Great idea!

File: go/pkg/cli/show.go (hunk 2 above)
Comment (bfirsh) [reply go/pkg/cli/show.go:166]: Thanks!

File: go/pkg/cli/show.go (hunk 1 above)
Comment (bfirsh) [reply go/pkg/cli/show.go:182]: good idea

**Comment by bfirsh** [comment]
Thanks for review! Fixed issues.

**Review by andreasjansson — APPROVED** [review]

**Pull request merged**